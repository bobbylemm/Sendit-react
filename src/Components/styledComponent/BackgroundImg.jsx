import styled from 'styled-components';

const BackgroungImg = styled.main`
  flex: 1;
  background-position: center center;
  background-size: cover;
  overflow: auto;
`;

export default BackgroungImg;
