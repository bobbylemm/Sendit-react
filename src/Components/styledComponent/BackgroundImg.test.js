import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import BackgroundImg from './BackgroundImg';

describe('testing the button element', () => {
  it('should mount button', () => {
    const tree = renderer.create(<BackgroundImg />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).toHaveStyleRule('background-position', 'center center');
  });
});
