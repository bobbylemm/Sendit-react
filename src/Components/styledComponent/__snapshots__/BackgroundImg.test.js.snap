// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the button element should mount button 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  background-position: center center;
  background-size: cover;
  overflow: auto;
}

<main
  className="c0"
/>
`;
